### ISSUE TITLE
Accessibility - Methodology Screen - TalkBack does not Read Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#28]
TalkBack does not read the links as a link on this screen. Not sure if the little bell sounds is what is supposed to tell a person that certain text is a link.

**Comment by aquilent-dgemoets** [comment]
See discussion for #26.

**Comment by jfc3-dol** [comment]
Okay, this could be confusing for those using iOS at work and then Android at home or vice versa.
